miden-lib/asm/kernels/transaction/main.masm

```diff
@@ -5,6 +5,37 @@ use.miden::kernels::tx::memory
 use.miden::kernels::tx::note
 use.miden::kernels::tx::prologue
 
+# TRACES
+# =================================================================================================
+
+# Trace emitted to signal that an execution of the transaction prologue has started.
+const.PROLOGUE_START=131072
+# Trace emitted to signal that an execution of the transaction prologue has ended.
+const.PROLOGUE_END=131073
+
+# Trace emitted to signal that the notes processing has started.
+const.NOTES_PROCESSING_START=131074
+# Trace emitted to signal that the notes processing has ended.
+const.NOTES_PROCESSING_END=131075
+
+# Trace emitted to signal that the note consuming has started.
+const.NOTE_CONSUMING_START=131076
+# Trace emitted to signal that the note consuming has ended.
+const.NOTE_CONSUMING_END=131077
```

Comment:
nit: I would name these `NOTE_EXECUTION_START` and `NOTE_EXECUTION_END` (here and in other relevant places).